aikau/src/main/resources/alfresco/search/FacetFilter.js

```diff
@@ -187,14 +203,21 @@ define(["dojo/_base/declare",
        *
        * @instance
        */
-      onToggleFilter: function alfresco_search_FacetFilter__onToggleFilter(/*jshint unused:false*/ evt) {
-         if (this.applied)
+      onToggleFilter: function alfresco_search_FacetFilter__onToggleFilter(evt) {
+         if (evt.target !== this.labelNode && !this.fullWidthClick)
```

Comment:
This is one of the problems with having negative conditions in an if-else ... more difficult to understand than positive. So, you will change the filters if you've clicked on the label node or full width is turned on. Would prefer to see it that way round.